modules/end-user-guide/examples/snip_che-ide-default-yaml-file.adoc

```diff
@@ -0,0 +1,7 @@
+:_content-type: SNIPPET
+
+[NOTE]
+====
+The default `devfile.yaml` file featuring settings for the supported IDEs is available link:https://github.com/eclipse-che/che-plugin-registry/blob/main/che-editors.yaml[here].
+====
```

Comment:
Since this snippet now goes into the TIP content, you can remove the admonition formatting from here:
```suggestion
* The default `devfile.yaml` file featuring settings for the supported IDEs is available link:https://github.com/eclipse-che/che-plugin-registry/blob/main/che-editors.yaml[here].
```